battle.cpp

```diff
@@ -1050,7 +1090,7 @@ int Game::RunBattle(ARegion * r,Unit * attacker,Unit * target,int ass,
 			}
 		}
 	}
-	result = b->Run(r,attacker,&atts,target,&defs,ass, &regions );
+	result = b->Run(events, r,attacker,&atts,target,&defs,ass, &regions );
```

Comment:
just nit - lets try to follow either no space or add space between variables convention. Not a big deal.